gateways/core/calendar/config.go

```diff
@@ -43,6 +45,23 @@ type calSchedule struct {
 	// the combination of these rules and dates combine to form a set of date times.
 	// NOTE: functionality currently only supports EXDATEs, but in the future could be expanded.
 	Recurrence []string `json:"recurrence,omitempty"`
+
+	// Timezone in which to run the schedule
+	// +optional
+	Timezone string `json:"timezone,omitempty"`
+
+	// UserPayload will be sent to sensor as extra data once the event is triggered
+	// +optional
+	UserPayload string `json:"userPayload,omitempty"`
```

Comment:
any reason this is a string and not []byte ?

Reply:
Most probably user will put JSON string as UserPayload.